incubator/group/types.go

```diff
@@ -49,9 +53,19 @@ type DecisionPolicyResult struct {
 type DecisionPolicy interface {
 	orm.Persistent
 	orm.Validateable
+	GetThreshold() sdk.Dec
+	GetTimout() types.Duration
```

Comment:
We actually don't want these on the `DecisionPolicy` interface. They are just relevant to `ThresholdDecisionPolicy` and might not apply to other `DecisionPolicy`'s.

Reply:
As discussed, it makes sense.

Just a question though, in https://github.com/regen-network/cosmos-modules/blob/marie/group-proto-any/incubator/group/keeper.go#L514-L527, `GetThreshold()` and `GetTimout()` are used. Should I just cast to `*ThresholdDecisionPolicy` and return an error if it's not the underlying type for now?

`Timeout` should be part of a proposal (https://github.com/regen-network/cosmos-modules/blob/marie/group-proto-any/incubator/group/types.proto#L247) so should `GetTimeout` actually stay on the `DecisionPolicy` interface level?

Reply:
> Just a question though, in https://github.com/regen-network/cosmos-modules/blob/marie/group-proto-any/incubator/group/keeper.go#L514-L527, `GetThreshold()` and `GetTimout()` are used. Should I just cast to `*ThresholdDecisionPolicy` and return an error if it's not the underlying type for now?

We want to avoid tying the keeper logic too much to a concrete implementation. This is all validation logic, so instead we should have some sort of `Validate()` method on `Policy` that takes the parameters needed to do this checking.
 
> `Timeout` should be part of a proposal (https://github.com/regen-network/cosmos-modules/blob/marie/group-proto-any/incubator/group/types.proto#L247) so should `GetTimeout` actually stay on the `DecisionPolicy` interface level?

I think `GetTimeout` can stay on `DecisionPolicy`